gramps/plugins/webreport/basepage.py

```diff
@@ -222,6 +222,9 @@ def __init__(self, report, the_lang, the_title, gid=None):
             self.rlocale = report.set_locale(the_lang)
         else:
             self.rlocale = report.set_locale(report.options["trans"])
+        self.dir = "ltr"
```

Comment:
A list of [RTL locales](https://github.com/gramps-project/gramps/blob/master/gramps/gen/utils/grampslocale.py#L85) is maintained in the GrampsLocale class.

We could access this list with code like:

`self.dir = "rtl" if self.rlocale.rtl_locale else "ltr"`

Reply:
Thanks for that information. Applied.